lib/mobility/plugins/dirty.rb

```diff
@@ -20,6 +20,8 @@ module Plugins
     module Dirty
       extend Plugin
 
+      default true
```

Comment:
This causes a lot of problems in specs, because the specs currently run with the plugin included but disabled by default. I've explicitly set it to `false` in specs now so it can still be included, but really specs should only include this and other auto-enabled plugins when they are actually being tested; otherwise, they shouldn't be included in the first place.